utils.py

```diff
@@ -604,13 +644,24 @@ def is_valid_language_code(language_code):
 
 
 def unescape_encoded_uri_component(escaped_string):
-    """Unescape a string that is encoded with encodeURIComponent."""
+    """Unescape a string that is encoded with encodeURIComponent.
+
+    Args:
+        escaped_string: str. Passing escaped string
```

Comment:
Again, the docstring should describe the variable. Here also, it is describing an action. Think about, what the parameter ( esaceped_string) is.